app/frontend/build/webpack.base.conf.js

```diff
@@ -21,12 +21,6 @@ const createLintingRule = () => ({
 
 module.exports = {
   context: path.resolve(__dirname, '../'),
-  entry: {
```

Comment:
I think we need this in all three our environments, so I'd rather not re-produce it in dev, test and prod.

Reply:
If we don't define which app to build for the dev environment, it will try to serve all of them. Only the prod build needs to include all apps.

Reply:
It needs the entry-points defined though though, or else webpack won't build the bundles that python uses. Like what you've done in webpack.dev.conf.js for submissions.